src/main/kotlin/com/hibob/academy/dao/OwnersDao.kt

```diff
@@ -0,0 +1,50 @@
+package com.hibob.academy.dao
+
+import org.jooq.DSLContext
+import org.jooq.Record
+import org.jooq.RecordMapper
+import org.springframework.stereotype.Component
+import jakarta.inject.Inject
+
+@Component
+class OwnerDao @Inject constructor(private val sql: DSLContext) {
+
+    private val ownerTableInstance = OwnerTable.ownerInstance
+
+    private val ownerMapper = RecordMapper<Record, Owner> { record ->
+        Owner(
+            //id = record[ownerTableInstance.id],
+            name = record[ownerTableInstance.name],
+            companyId = record[ownerTableInstance.companyId],
+            employeeId = record[ownerTableInstance.employeeId]
+        )
+    }
+
+    fun insertOwner(name: String, companyId: Long, employeeId: String) {
+        sql.insertInto(ownerTableInstance)
+            .columns(ownerTableInstance.name, ownerTableInstance.companyId, ownerTableInstance.employeeId)
+            .values(name, companyId, employeeId)
+            //.onConflict(ownerTableInstance.name, ownerTableInstance.companyId, ownerTableInstance.employeeId)
+            //.doNothing()
+            .execute()
```

Comment:
Why did you comment out?
What will happen when you try to insert the same owner twice?